pages/about.js

```diff
@@ -28,10 +29,7 @@ const About = () => (
       </div>
       <DiversityAndInclusionSection />
       <ExecutiveTeamSection />
-      <LogoSection
```

Comment:
The index page and the about page shared the same component (`LogoSection`). But the layout is significantly different making this shared component really hard to reuse.

I chose to have 2 sections: one for the index and one for the about page. The only shared piece is the image container.